vos/vos/vos.py

```diff
@@ -112,7 +112,32 @@ class SortNodeProperty(Enum):
     )
 MAGIC_GLOB_CHECK = re.compile('[*?[]')
 
+
+logging.getLogger("requests").setLevel(logging.ERROR)
+
+
+def _rename_vospace_resource():
+    # temporary function to deal with renaming of the
+    # ivo://cadc.nrc.ca/vospace to ivo://cadc.nrc.ca/vault
+    if os.path.exists(_CONFIG_PATH):
+        try:
+            config_content = open(_CONFIG_PATH, 'r').read()
+            if 'ivo://cadc.nrc.ca/vospace' in config_content:
+                config_content = config_content.replace(
+                    'ivo://cadc.nrc.ca/vospace',
+                    'ivo://cadc.nrc.ca/vault')
+                open(_CONFIG_PATH, 'w').write(config_content)
+                warnings.warn(
```

Comment:
I'm not sure this requires a warning when it runs?  this would confuse users more than inform them.